API_GUIDE.md

```diff
@@ -152,6 +160,25 @@ See the
 [full multiprocessing example](https://github.com/pytorch/xla/blob/master/test/test_train_mp_mnist.py)
 for more on training a network on multiple XLA devices with multi-processing.
 
+### Running on Multiple XLA Hosts
+Multi-host setup for different accelerators can be very different. This doc will talk about the device independent bits of multi-host training and will use the TPU + PJRT runtime(currently available on 1.13 and 2.x releases) as an example.
+
+Let's assume you have the above mnist example from above section in a `train_mnist_xla.py`. If it is a single host multi device training, you would run it like
+
+```
```

Comment:
nit: maybe mention the user need to ssh into the TPU VM and run the command there?

Reply:
good point, I forgot to mentioned where to run these commands.